e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/MetadataTest.groovy

```diff
@@ -29,24 +29,23 @@ class MetadataTest extends Specification {
 
         when:
         def inboundMetadataResponse = metadataClient.get(inboundSubmissionId, true)
-        def outboundMetadataResponse = metadataClient.get(outboundSubmissionId, true)
```

Comment:
We support both the inbound or outbound ID being passed in, so we need to keep this test code here.